### PR TITLE
Use mustExec to simplify SQL execution in test cases

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

There are many duplicate code for executing SQLs and exitting on error.

### What is changed and how it works?

Reuse `mustExec`.

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes

### CODE REVIEW

**Comment by july2993** [comment]
@suzaku pls fix ci

**Comment by suzaku** [comment]
@july2993 一直出现 pd server timeout，有遇到过这种情况吗？

**Comment by july2993** [comment]
> @july2993 一直出现 pd server timeout，有遇到过这种情况吗？

you can try specify release3.0, maybe master branch of tidb/pd is broken againg.

**Comment by suzaku** [comment]
/run-integration-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by suzaku** [comment]
@july2993 It's OK when I specify the 3.0 version.

File: tests/dailytest/case.go [hunk 1]
```diff
@@ -233,15 +233,9 @@ func RunCase(src *sql.DB, dst *sql.DB, schema string) {
...
-		if err != nil {
-			log.S().Fatal(err)
-		}
+		mustExec(src, "create table uindex(id int primary key, a1 int unique)")
 
-		_, err = src.Exec("insert into uindex(id, a1) values(1, 10),(2,20)")
-		if err != nil {
-			log.S().Fatal(err)
-		}
+		mustExec(src, "insert into uindex(id, a1) values(1, 10),(2,20)")
```
Comment (lichunzhu) [inline tests/dailytest/case.go]:
```suggestion
		mustExec(src, "insert into uindex(id, a1) values(1, 10), (2, 20)")
```

**Comment by lichunzhu** [comment]
/run-integration-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Pull request merged**